backend/attendance-service/src/main/java/org/egov/validator/AttendanceLogServiceValidator.java

```diff
@@ -109,81 +112,81 @@ private void validateAttendees(AttendanceLogRequest attendanceLogRequest) {
          */
 
 
-        if("TRUE".equalsIgnoreCase(config.getIndividualServiceIntegrationRequired())){
+        if ("TRUE".equalsIgnoreCase(config.getIndividualServiceIntegrationRequired())) {
             //TODO
             // For now throwing exception. Since individual service is under discussion.
-            throw new CustomException("INTEGRATION_UNDERDEVELOPMENT","Individual service integration is under development");
+            throw new CustomException("INTEGRATION_UNDERDEVELOPMENT", "Individual service integration is under development");
         }
 
         // Fetch all attendees for given register_id.
         String registerId = attendanceLogRequest.getAttendance().get(0).getRegisterId();
         List<IndividualEntry> fetchAttendanceAttendeeLst = fetchAllAttendeesEnrolledInARegister(registerId);
 
         // Convert the fetched Attendee List into a Map with individualId as key and corresponding Attendee list as value.
-        Map<UUID,List<IndividualEntry>> attendanceAttendeeListMap = fetchAttendanceAttendeeLst
-                                                                            .stream()
-                                                                            .collect(Collectors.groupingBy(IndividualEntry::getIndividualId));
+        Map<UUID, List<IndividualEntry>> attendanceAttendeeListMap = fetchAttendanceAttendeeLst
```

Comment:
We don't need type `UUID` anywhere. You can use String.

Reply:
Change implemented